### ISSUE TITLE
Getter function is skipped without error if it has a required argument

### ISSUE THREAD

**Issue by romlok** [godotengine/godot#19306]
**Godot version:**
3.0.2 (Steam)

**OS/device including version:**
Linux x64; X11; Debian Testing

**Issue description:**
If a GDScript variable is defined to use a getter function that has a required parameter (ie. one with no default value), the getter will not be called, and no error gets output.

I would expect an error to be logged in such a case.

**Steps to reproduce:**
```gdscript
class MyClass:
	var my_val = "default" setget ,my_getter
	
	func my_getter(oops):
		return "gotten"

func _ready():
	var foo = MyClass.new()
	print(foo.my_val)
```
Running this will print `default` rather than `gotten`, but not show any error message.

**Comment by mrcdk** [comment]
Getters functions don't have parameters. http://docs.godotengine.org/en/latest/getting_started/scripting/gdscript/gdscript_basics.html#setters-getters

This works:
```
class MyClass:
	var my_val = "default" setget ,my_getter
	
	func my_getter():
		return "gotten"

func _ready():
	var foo = MyClass.new()
	print(foo.my_val)
```

**Comment by romlok** [comment]
Ah, I guess my example's conclusion wasn't clear originally (now edited).
The issue isn't that the code doesn't work, but that the invalid code doesn't produce an error or warning message!

**Comment by swarnimarun** [comment]
This actually works, 

```swift
var val = "default" setget  ,get_me

func get_me(num = 0):
    return "awesome"
```

**Comment by bojidar-bg** [comment]
The problem is not that the sample code does not work, the problem is that it does not produce an error.

**Comment by vnen** [comment]
Solved by #19264. It will now show an error in-editor if the getter or setter is invalid.
